--- a/src/service/service.js
+++ b/src/service/service.js
@@ -35,7 +35,7 @@
 		const token = localStorage.getItem("access_token")
 		const rtoken = localStorage.getItem("refresh_token")
 		if (token) {
-			config.headers.Authorization = token
+			config.headers.Authorization = "Bearer " + token
 			store.commit("SET_ACCESS_TOKEN", token);
 			store.commit("SET_REFRESH_TOKEN", rtoken);
 		}
@@ -81,8 +81,8 @@
 
 						store.commit("SET_ACCESS_TOKEN", tokenRes.data.data.access_token);
 						store.commit("SET_REFRESH_TOKEN", tokenRes.data.data.refresh_token);
-						originalConfig.headers.Authorization = tokenRes.data.data.access_token
-						instance.defaults.headers.Authorization = tokenRes.data.data.access_token
+						originalConfig.headers.Authorization = "Bearer " + tokenRes.data.data.access_token
+						instance.defaults.headers.Authorization = "Bearer " + tokenRes.data.data.access_token
 						isRefreshing = false
 						return tokenRes.data.data.access_token
 					} else {
@@ -102,7 +102,7 @@
 			return new Promise(resolve => {
 				requests.push((token) => {
 					originalConfig.headers = {}
-					originalConfig.headers.Authorization = token
+					originalConfig.headers.Authorization = "Bearer " + token
 					resolve(instance(originalConfig))
 				})
 			})
--- a/src/components/filebrowser/FilePanel.vue
+++ b/src/components/filebrowser/FilePanel.vue
@@ -354,7 +354,7 @@
 			permanentErrors: [404, 409, 415, 500, 501],
 			allowDuplicateUploads: true,
 			headers: {
-				Authorization: this.token,
+				Authorization: this.token ? "Bearer " + this.token : this.token,
 			},
 			query: (file) => {
 				return {
@@ -400,7 +400,7 @@
 	watch: {
 		"$store.state.access_token": {
 			handler(val) {
-				this.options.headers.Authorization = val;
+				this.options.headers.Authorization = val ? "Bearer " + val : val;
 			},
 			deep: true,
 		},
--- a/src/components/filebrowser/uploader/components/uploader.vue
+++ b/src/components/filebrowser/uploader/components/uploader.vue
@@ -92,7 +92,7 @@
 			this.fileList = this.uploader.fileList
 			this.$api.sys.getVersion().then(res => {
 				if (this.autoStart && res.status === 200) {
-					this.uploader.opts.headers.Authorization = this.$store.state.access_token || localStorage.getItem("access_token")
+					this.uploader.opts.headers.Authorization = "Bearer " + (this.$store.state.access_token || localStorage.getItem("access_token"))
 					this.uploader.upload()
 				}
 			})
--- a/src/components/wallpaper/WallpaperModal.vue
+++ b/src/components/wallpaper/WallpaperModal.vue
@@ -98,7 +98,7 @@
 		this.uploader.on('filesSubmitted', () => {
 			this.isUpLoading = true
 			this.$api.sys.getVersion().then(res => {
-				this.uploader.opts.headers.Authorization = this.$store.state.access_token || localStorage.getItem("access_token")
+				this.uploader.opts.headers.Authorization = "Bearer " + (this.$store.state.access_token || localStorage.getItem("access_token"))
 				this.uploader.upload()
 			})
 		})
